### DISCUSSION TITLE
Remove all Oracle vestiges?

### DISCUSSION THREAD

**Discussion by wokket** [grate-devs/grate#50] [Ideas]
I have no intention of trying to make Oracle support work, but we have a bunch of classes filled with `NotImplementedExceptions` and Oracle dependencies to update/download etc.

I propose we remove all ORacle stuff from the solution until we can find someone willing to own it (by contributing code, $$$ or beers to make it happen).

**Comment by erikbra** [comment]
Yea... I see your point. Thing is, making it work is dead easy (famous last words) if we can just get a f*cking connection string that works. I tried to read on and on how to create an Oracle connection string without creating a TNSNAMES.ORA file and whatnot, but couldn't figure it out.

I'll think a bit. Maybe we'll leave it commented out, or delete it. I'll ponder a bit. I just want to show how easy it is to implement it, ehn we go public :)

**Comment by erikbra** [comment]
FYI, I've started working on Oracle support. Oracle has some special needs, but I think I'm getting there

**Comment by erikbra** [comment]
Oracle support is implemented and done. Is there any way to close/remove disscussions, I wonder...
